test/io-tests/test_io.py

```diff
@@ -209,6 +209,22 @@ def test_expected_config_from_environment():
     assert dumpconfig(env=env) == expected
 
 
+def test_expected_config_from_db_settings(defaultenv):
+    "Config should be overriden from database settings"
+
+    env = defaultenv
+    env["PGRST_DB_SETTINGS_PREFIX"] = "instance"
+    expected = (CONFIGSDIR / "db-settings-prefix.config").read_text()
```

Comment:
This file would better be in the `expected` subdir. I suggest you just re-use one of the files in this directory. The `no-defaults.config` would fit, because it's exactly that. If you change the fixtures to those values, you can use it.

Reply:
Or, alternatively, if that's easier, move the file you have to `expected`. You might change some condition for one of the others tests that loads it's fixtures from the list of files in `expected`, but that should be straight forward.

Reply:
Done! I've put the config in expected.